src/main/java/com/example/TwoRoomsBoom/models/Player.java

```diff
@@ -0,0 +1,53 @@
+package com.example.TwoRoomsBoom.models;
+
+import java.util.Set;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Builder.Default;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Player {
+    private String name;
+    private Role role;
+    private Set<String> conditions;
+    private Integer room;
+    @Default
+    private boolean isLeader = false;
+    private boolean isHost;
+    
+    public void changeRoom() {
```

Comment:
Thinking about this now, should the player have this action? Or should the Room Class be in control of changing the rooms of players?

Reply:
Good point, Room Class should probably handle exchanging players, not the Player Class.